meta/group-info/es.yaml

```diff
@@ -1,34 +1,29 @@
-outdated_since: 0522bc4c323f9a2b0c89a6b9582109a0608c6287
-outdated_translation: true
-
 separator: ", "
 alumni:
   roles:
-    academy: fundador de osu!academy
+    academy: fundador de la osu!academy
     admin: administración
     chat: moderador del chat
     dev: osu!dev
     media: creador de osu!media
     support: equipo de soporte
-    tournaments: gestor de torneos
+    tournaments: gestión de torneos
```

Comment:
Because this section is about roles I wouldn't change `gestor` to `gestión` as the title of the role is correct as is. For the areas the update is correct.

Reply:
aaaaa

Reply:
would it be the same in admin?

Reply:
would it be the same in admin?

Reply:
Yes. Nice catch.